tests/phpunit/Compat/WC_Stripe_Email_Failed_Refund_Test.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace WooCommerce\Stripe\Tests\Compat;
+
+use WC_Email;
+use WC_Stripe_Email_Failed_Refund;
+use WooCommerce\Stripe\Tests\Helpers\WC_Helper_Order;
+use WP_UnitTestCase;
+
+/**
+ * Class WC_Stripe_Email_Failed_Refund_Test
+ *
+ * @package WooCommerce/Stripe/WC_Stripe_Email_Failed_Refund
+ *
+ * Class WC_Stripe_Email_Failed_Refund tests.
+ */
+class WC_Stripe_Email_Failed_Refund_Test extends WP_UnitTestCase {
+	/**
+	 * Tests for the `trigger` method.
+	 *
+	 * @param bool   $is_enabled     Whether the email is enabled.
+	 * @param string $recipient      The recipient email address.
+	 * @param bool   $expect_to_send Whether the email is expected to be sent.
+	 * @return void
+	 *
+	 * @dataProvider provide_test_trigger
+	 */
+	public function test_trigger( $is_enabled, $recipient, $expect_to_send ) {
+		$order = WC_Helper_Order::create_order();
+
+		$email = $this->getMockBuilder( WC_Stripe_Email_Failed_Refund::class )
+			->disableOriginalConstructor()
+			->onlyMethods( [ 'send', 'is_enabled', 'get_recipient', 'get_content', 'get_template_params' ] )
+			->getMock();
+
+		$email->expects( $expect_to_send ? $this->once() : $this->never() )
+			->method( 'send' );
+
+		$email->expects( $this->once() )
+			->method( 'is_enabled' )
+			->willReturn( $is_enabled );
+
+		$email->expects( $recipient ? $this->exactly( 2 ) : ( $is_enabled ? $this->once() : $this->never() ) )
+			->method( 'get_recipient' )
+			->willReturn( $recipient );
+
+		$email->expects( $is_enabled && $recipient ? $this->once() : $this->never() )
+			->method( 'get_content' )
+			->willReturn( 'Email content' );
+
+		$email->trigger( $order->get_id(), $order );
+	}
+
+	/**
+	 * Provider for the `test_trigger` method.
+	 *
+	 * @return array
+	 */
+	public function provide_test_trigger() {
+		return [
+			'not enabled'  => [
+				'is enabled'     => false,
+				'recipient'      => '',
```

Comment:
I think this test should include a recipient so we are more likely to verify that it is not sent based on the enabled flag.
```suggestion
				'recipient'      => 'test@example.com',
```